test/e2e/run_test.go

```diff
@@ -216,4 +216,27 @@ var _ = Describe("Podman run", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman run with secrets", func() {
+		containersDir := filepath.Join(podmanTest.TempDir, "containers")
+		err := os.MkdirAll(containersDir, 0755)
+		Expect(err).To(BeNil())
+
+		secretsDir := filepath.Join(podmanTest.TempDir, "rhel", "secrets")
+		err = os.MkdirAll(secretsDir, 0755)
+		Expect(err).To(BeNil())
+
+		mountsFile := filepath.Join(containersDir, "mounts.conf")
+		mountString := secretsDir + ":/run/secrets"
+		err = ioutil.WriteFile(mountsFile, []byte(mountString), 0755)
+		Expect(err).To(BeNil())
+
+		secretsFile := filepath.Join(secretsDir, "test.txt")
+		secretsString := "Testing secrets mount. I am mounted!"
+		err = ioutil.WriteFile(secretsFile, []byte(secretsString), 0755)
+		Expect(err).To(BeNil())
+
+		session := podmanTest.Podman([]string{"--default-mounts-file", mountsFile, "run", "--rm", ALPINE, "cat", "/run/secrets/test.txt"})
```

Comment:
Tests will not pass. Need to add the `default-mounts-file` flag to main.go and pass it in to where the secretes are being mounted.